keyring/backends/macOS/__init__.py

```diff
@@ -17,8 +18,8 @@
 class Keyring(KeyringBackend):
     """macOS Keychain"""
 
-    keychain = os.environ.get('KEYCHAIN_PATH')
-    "Path to keychain file, overriding default"
+    # keychain = os.environ.get('KEYCHAIN_PATH')
+    # "Path to keychain file, overriding default. function param "
```

Comment:
Commented code is usually a bad smell. Better would be to actually supply the attribute using the default value.